saw-core/prelude/Prelude.sawcore

```diff
@@ -117,6 +117,14 @@ data Eq (t : sort 1) (x : t) : t -> Prop where {
     Refl : Eq t x x;
   }
 
+-- The version of Eq in Prop
+-- FIXME: this should replace the predicative Eq when we are sure everything
+-- works ok with it
+data EqP (t : sort 1) (x : t) : t -> Prop where {
+    ReflP : EqP t x x;
+  }
+
+
```

Comment:
We don't want this; #154 changed the sort of `Eq` so that you can just use `Eq`, and this would be an exact duplicate.

Reply:
Should be gone.